Sources/Encodings/MobUri.swift

```diff
@@ -28,24 +33,29 @@ public enum MobUri {
     }
 
     public static func encode(_ publicAddress: PublicAddress) -> String {
-        encode(.publicAddress(publicAddress))
+        logger.info("")
+        return encode(.publicAddress(publicAddress))
     }
 
     public static func encode(_ paymentRequest: PaymentRequest) -> String {
-        encode(.paymentRequest(paymentRequest))
+        logger.info("")
+        return encode(.paymentRequest(paymentRequest))
     }
 
     public static func encode(_ transferPayload: TransferPayload) -> String {
-        encode(.transferPayload(transferPayload))
+        logger.info("")
+        return encode(.transferPayload(transferPayload))
     }
 
     static func encode(_ payload: Payload) -> String {
-        "\(McConstants.MOB_URI_SCHEME)://\(payload.uriPath)"
+        logger.info("")
+        return "\(McConstants.MOB_URI_SCHEME)://\(payload.uriPath)"
```

Comment:
`payload` is sensitive.

Reply:
payload isn't being logged here, just a blank log statement at the start of the encode function

Reply:
Oops! You're right